source/codegen/service_helpers.py

```diff
@@ -55,6 +55,8 @@ def create_param(parameter):
         return f'{type} {name}'
 
 def python_to_c(enum):
```

Comment:
note: If the enum is empty, return default value. This could happen now since we could have empty `AttributeValues` enums for specific types if there are no attributes of that type. Same check is done for some of the helpers below as well.